modules/priceFloors.js

```diff
@@ -501,7 +507,13 @@ export function handleFetchResponse(fetchResponse) {
     floorResponse = fetchResponse;
   }
   // Update the global floors object according to the fetched data
-  _floorsConfig.data = parseFloorData(floorResponse, 'fetch') || _floorsConfig.data;
+  const fetchData = parseFloorData(floorResponse, 'fetch');
+  if (fetchData) {
+    // set .data to it
+    _floorsConfig.data = fetchData;
+    // set skipRate override if necessary
+    _floorsConfig.skipRate = utils.isNumber(fetchData.skipRate) ? fetchData.skipRate : _floorsConfig.skipRate;
```

Comment:
We were not honoring fetch level skip rates, which is a bug. Need to get this out asap.

Added a test to confirm.